examples/mnist/mnist_lib.py

```diff
@@ -27,6 +27,8 @@
 
 import numpy as onp
 
+import tensorflow as tf
+tf.config.experimental.set_visible_devices([], "GPU")
```

Comment:
same comment as above - needs to go in main file, not lib file.

Reply:
ditto.